<?xml version="1.0" encoding="UTF-8"?>
<launch>
<group ns="X1">
  <node pkg="ground_finder" type="traversability_mapping" name="traversability_mapping" output="screen">
  	<!-- <remap from="octomap_binary" to="merged_map"/> -->
	<remap from="octomap_binary" to="octomap_binary"/>
	<remap from="edt" to="edt"/>

  	<!-- Params -->
  	<param name="update_rate" value="2.0"/>
  	<param name="min_cluster_size" value="50"/>
    <!-- 0.95 filters out stairs in the EC Basement -->
  	<param name="normal_z_threshold" value="0.92"/>
	<param name="sensor_range" value="15.0"/>
	<!-- <param name="bbx_xy" value="8.0"/> -->
	<!-- <param name="bbx_z" value ="3.0"/> -->
  </node>
</group>
</launch>